libbeat/autodiscover/factoryadapter.go

```diff
@@ -26,20 +26,20 @@ import (
 	"github.com/elastic/beats/libbeat/common/bus"
 )
 
-// AutodiscoverAdapter for Metricbeat modules
-type AutodiscoverAdapter struct {
+// FactoryAdapter for Metricbeat modules
```

Comment:
you will need to update the docs here (Metricbeat references), probably explain a little bit how this takes a factory and give you an autodiscover adapter